core/examples/luigi-sample-angular/src/assets/extendedConfiguration.js

```diff
@@ -170,6 +170,11 @@ var projectDetailNavProviderFn = function(context) {
           }
         ]
       },
+      {
+        absolutePath: '/settings',
+        label: 'Go to absolute path',
+        viewUrl: '/sampleapp.html#/settings'
```

Comment:
The `viewUrl` is ignored here. This feature redirects to an existing view, doesn't create a new one.